plugins/ccgx/ccgx.quirk

```diff
@@ -10,15 +10,16 @@ GType = FuCcgxHpiDevice
 
 [DeviceInstanceId=USB\VID_04B4&PID_521A&SID_1F00&APP_6D64]
 ImageKind = dual-asymmetric
+Name = ThinkPad USB-C Dock Gen2 PD
```

Comment:
"ThinkPad USB-C Dock Gen2 PD Controller" is better to get align with Hybrid Dock Name: "ThinkPad USB-C Dock Hybrid PD Controller"

Reply:
Fixed.